core/src/test/java/org/infinispan/test/TestingUtil.java

```diff
@@ -1447,9 +1434,9 @@ public static ObjectName getJGroupsChannelObjectName(EmbeddedCacheManager cacheM
       GlobalConfiguration cfg = cacheManager.getCacheManagerConfiguration();
       try {
          return new ObjectName(String.format("%s:type=channel,cluster=%s,manager=%s",
-                                             cfg.jmx().domain(),
-                                             ObjectName.quote(cacheManager.getClusterName()),
-                                             ObjectName.quote(cfg.cacheManagerName())));
+               cfg.jmx().domain(),
+               ObjectName.quote(cacheManager.getClusterName()),
+               ObjectName.quote(cfg.cacheManagerName())));
```

Comment:
-1, I really like the aligned method parameters (and chained method calls)
Plus, you aligned the parameter descriptions in the javadoc, which seems less useful to me :)

Reply:
I've reverted all changes.